docs/how-to/ChIP_seq_genomic_ranges_tutorial.ipynb

```diff
@@ -0,0 +1,2733 @@
+{
```

Comment:
<p>How difficult would it be to plot a gene and associated peak? Like with <code>bioframe.plot_intervals</code> or <code>gos</code> ?</p><p> </p><p>Not a requirement, but I think could be nice! </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/genomic-features/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I'm planning on doing this in the single-cell 10X Visium Multiome (atac-seq was mentioned in one of the issues I think so wanted to tackle that specifically) tutorial I'm working on now! Should hopefully have this for you by Monday, if all things go to plan

Reply:
<p>Monday may have been ambitious but I'm working on this*</p>